keyboards/puck/keymaps/default/keymap.c

```diff
@@ -0,0 +1,64 @@
+#include "puck.h"
```

Comment:
You can replace this with the following

```suggestion
#include QMK_KEYBOARD_H
```